src/rv32_template.c

```diff
@@ -107,6 +107,11 @@
  * |                                | store the result into dst.             |
  * | cond, src;                     | set condition if (src)                 |
  * | end;                           | set the end of condition if (src)      |
+ * | parsebt;                       | parse the branch table of indirect     |
```

Comment:
Avoid using "parse", which might confuse. The intention was related to branch prediction. How about something like `predict`?

Reply:
Ok, I think predict is better.